h2/src/main/org/h2/command/dml/Insert.java

```diff
@@ -46,6 +47,7 @@
     private boolean sortedInsertMode;
     private int rowNumber;
     private boolean insertFromSelect;
+    private TableFilter sourceTableFilter;
```

Comment:
please add a comment like // for MERGE..USING

Reply:
done